testdata/backup-management/mongodb/setup-replica-and-pbm-local.sh

```diff
@@ -1,69 +1,16 @@
 #!/bin/bash
 
-set -e
+PWD=$(pwd) docker-compose -f docker-compose-mongo-replica-local.yml up -d mongors1 mongors2 mongors3 --build
+sleep 5
+docker exec -it mongors1 bash /opt/pbm/configure-replica.sh
 
-PWD=$(pwd) docker-compose -f docker-compose-mongo-replica-local.yml up -d
+PWD=$(pwd) docker-compose -f docker-compose-mongo-replica-local.yml up -d pbm-agent-rs1  pbm-agent-rs2 pbm-agent-rs3 pmm-client --build
 
-cat > setup-replica.js <<EOF
-rs.initiate(
```

Comment:
I couldn't figure how to do authentication in the generated javascript file, so I've used a bash script for this instead (`configure-replica.sh`).